test/test.js

```diff
@@ -6889,3 +6890,53 @@ exports['use'] = {
         test.done();
     }
 };
+
+exports['streams can be used as orchestrator tasks'] = function (test) {
+    var isReady = false;
+    var s = _();
+    runTask(function () { return s; }, function () {
+        test.ok(isReady, "task finished too early");
+        test.done();
+    });
+    setTimeout(function () {
+        isReady = true;
+        s.end();
+    }, 10);
+};
+
+exports['only gutless streams and pipelines are writable'] = function (test) {
```

Comment:
This test is missing a few constructor signatures (all not writable).

- `_('event', new EventEmitter())`
- `_(through())`
- `_(new Promise(function (res, rej) {})`
- `_(new Set())` - Iterable. Gate this behind a `global.Map` check (see line 688).
- `_(new Set().values())` - Iterator. Gate this behind a `global.Symbol` check.